docs/extensions/attr_list.md

```diff
@@ -158,6 +158,54 @@ assigned to the inline element (`<em>`) which immediately preceded it.
 
 Attribute lists may also be defined on table header cells (`<th>` elements) in the same manner.
 
+### Setting attributes on lists
+
+Unlike HTML there is no explicit start and end tag of a list in Markdown. A list implicitly begins when the first item
+is parsed and is implicitly closed after the last item. But the list items are the only places on which attributes
+can be set. To still be able to set attributes on the full list itself the following syntax can be used:
+
+
+```text
+1. Item1
+{^ start=4 }
+2. Item2
+{: #item2 }
+3. Item3
+{^ .inline }
+
+* Item1
+{^ .firstclass }
+* Item2
+{^ .secondclass }
+* Item3
+{^ #myid }
+```
+
+The use of `^` instead of `:` at the start of the attribute definition indicates that the attributes values are
```

Comment:
`:` isn't even needed anymore. It's a relict of the old syntax and is from what I know only still supported for backwards compatibility.
Either way, the new (and recommended) syntax is `{ <stuff> }`

Reply:
Fine, updated the docs